### PR TITLE
README: Refactor

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Closes https://github.com/coala-analyzer/coala-bears/issues/610

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
@hypothesist ive pushed a version thats consistent to the coala README so you can take a look if you want, i feel like we need to improve the Usage section a little and maybe others, so cmon, STRONG REVIEW!!!!!

File: README.rst [hunk 1]
```diff
@@ -1,33 +1,65 @@
...
+coala-bears
+-----------
 
 coala-bears is a Python package containing all the bears that are officially
-supported by coala. It features more than **65 bears** covering **35 languages**.
-Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_ that
-contains information about each bear, such as the languages it supports and what fixes
-it can apply to your code.
+supported by coala. It features more than **65 bears** covering
+**35 languages**. Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_
```
Comment (sims1253) [inline README.rst]: Those numbers are not correct anymore

File: README.rst [hunk 2]
```diff
@@ -1,33 +1,65 @@
...
+    :depth: 1
+    :backlinks: none
+
+-----
+
+============
+Installation
+============
+
+To install coala-bears, you first need Python 3 or higher. If you do not have it, it is recommended
```
Comment (hypothesist) [inline README.rst]: I'd mention 3.3+ since we don't support the others

File: README.rst [hunk 3]
```diff
@@ -1,33 +1,65 @@
...
+that you install Python3 >= 3.4 from `here <https://www.python.org/downloads/>`__.
```
Comment (hypothesist) [inline README.rst]: Ah didn't see this. Anyway, since already say `3.4, 3.5` with the `|PyPI|` I'm not sure we need this sent

File: README.rst [hunk 4]
```diff
@@ -1,33 +1,65 @@
...
+To install the latest development version run:
+
+.. code-block:: bash
+
+    $ pip3 install coala-bears --pre
+
+The latest code from the master branch is automatically deployed as the
+development version in PyPI.
+
+Be sure to use the latest pip, the default pip from Debian doesn't support our
```
Comment (hypothesist) [inline README.rst]: "Be sure to use the latest *version of* pip..."

Comment (hypothesist) [inline README.rst]: Also make change the comma into a `.` or a `;`

File: README.rst [hunk 5]
```diff
@@ -1,33 +1,65 @@
...
+dependency version number specifiers. You will have to use a
+`virtualenv <https://github.com/coala-analyzer/coala/wiki/FAQ#installation-is-failing-help>`__
```
Comment (hypothesist) [inline README.rst]: I'd rather have the link for `use a virtualenv` instead of just `virtualenv`

File: README.rst [hunk 6]
```diff
@@ -1,33 +1,65 @@
...
+in this case.
+
+|PyPI| |Windows| |Linux|
+
+-----
+
+===================
+Languages supported
```
Comment (hypothesist) [inline README.rst]: Capital S

File: README.rst [hunk 7]
```diff
@@ -1,33 +1,65 @@
...
+
+coala-bears
+-----------
 
 coala-bears is a Python package containing all the bears that are officially
-supported by coala. It features more than **65 bears** covering **35 languages**.
-Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_ that
-contains information about each bear, such as the languages it supports and what fixes
-it can apply to your code.
+supported by coala. It features more than **78 bears** covering
```
Comment (hypothesist) [inline README.rst]: Do we still want to say this? :P

As you mentioned before, this just increases the maintainence.

Comment (Adrianzatreanu) [inline README.rst]: yes

File: README.rst [hunk 8]
```diff
@@ -69,106 +98,86 @@ section before doing so.
 
 You can read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
 There you can learn how to easily write bears yourself! Be sure to let us know
-if you do so, then we'll be able to include it here and spread the word about it.
+if you do so, so we'll be able to include it here and spread the word about it.
```
Comment (hypothesist) [inline README.rst]: Add something like a link about how they can let us know

File: README.rst [hunk 9]
```diff
@@ -1,33 +1,62 @@
...
+coala-bears
+-----------
 
 coala-bears is a Python package containing all the bears that are officially
-supported by coala. It features more than **65 bears** covering **35 languages**.
-Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_ that
-contains information about each bear, such as the languages it supports and what fixes
-it can apply to your code.
+supported by coala. It features more than **78 bears** covering
+**36 languages**. Here is a `generated list <https://github.com/coala-analyzer/bear-docs/>`_
```
Comment (sils1297) [inline README.rst]: isn't it like 54?

Comment (Adrianzatreanu) [inline README.rst]: 54 what? languages??

Comment (hypothesist) [inline README.rst]: yes

File: README.rst (hunk 7 above)
Comment (abhsag24) [inline README.rst]: cant we automate? like count everything that has a Bear in the end? :P

Comment (abhsag24) [inline README.rst]: https://github.com/coala-analyzer/coala-bears/issues/618

File: README.rst [hunk 10]
```diff
@@ -69,106 +98,86 @@ section before doing so.
...
+=======
 
-We appreciate any help! Join us on one of:
+Feel free to contact us at our `Gitter channel <https://gitter.im/coala-analyzer/coala>`__, we'd be happy to help!
 
-- `gitter <https://gitter.im/coala-analyzer/coala/>`_
-- `#coala at freenode <webchat.freenode.net/?channels=coala>`_
-- `Telegram <https://telegram.me/joinchat/AuL-lwKZ8JLFZiI6SbtQVw>`_
+You can also drop an email at our
+`mailing list <https://github.com/coala-analyzer/coala/wiki/Mailing-Lists>`__.
```
Comment (sils1297) [inline README.rst]: Maybe add `If you are interested in commercial support, please contact us on the Gitter channel as well.` to encourage companies a bit? Also in the other readme.

File: README.rst (hunk 8 above)
Comment (Adrianzatreanu) [inline README.rst]: ignoring this as we talked about it.

File: README.rst (hunk 7 above)
Comment (Adrianzatreanu) [inline README.rst]: this is a cool enhancement, and should be added when done, but i dont its mandatory.

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/613/commits/416cbd7fd2b6a464872b993e6011ca4cfa72b037

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**